www/src/css/headline.css

```diff
@@ -8,7 +8,7 @@
   margin:0 0 0 10px
 }
 
-.b-headline .b-headline__store-link {
+.region-installed .b-headline__store-link {
```

Comment:
I probably miss some context here. Where is this `region-installed` class set?
Does it need to affect `b-headline__store-link`? Does `b-headline__store-link` have different styles in `region-installed`?